app/com/linkedin/drelephant/mapreduce/data/MapReduceApplicationData.java

```diff
@@ -148,6 +155,10 @@ public String getUsername() {
     return _username;
   }
 
+  public long getSubmitTime() {
```

Comment:
submitTime = submitTime we get from the job history server. All these values corresponds to the values in jobhistory server.